src/server/game/Entities/Unit/StatSystem.cpp

```diff
@@ -86,11 +86,31 @@ void Unit::UpdateDamagePhysical(WeaponAttackType attType)
     }
 }
 
+uint32 Unit::GetCreateVigor() const
+{
+    // Must be in "higher to lower" order
+    std::vector<uint32> vigorTalentSpellIds = {
```

Comment:
would prefer `static constexpr` array here. std::vector is pretty over kill for just looping over 3 values